packages/node_modules/@webex/plugin-meetings/src/reachability/request.js

```diff
@@ -24,7 +24,7 @@ getClusters = () => this.request({
   .then((res) => {
     const {clusters} = res.body;
 
-    LoggerProxy.logger.info(`ReachabilityRequest->getClusters#get clusters successful:${JSON.stringify(clusters)}`);
+    LoggerProxy.logger.info('ReachabilityRequest->getClusters#get clusters successful');
```

Comment:
How is this PII? This is the clusters the user got back.

Reply:
sorry i checked wrong response object, will revert this.